build_shared.sh

```diff
@@ -65,11 +65,15 @@ for platform in $PLATFORMS; do \
                      >"$KERNEL_TMP"/build.log 2>&1;
 
                 echo "Copying new kernel image ..."
-                cp "$KERNEL_TMP/arch/arm64/boot/Image.gz-dtb" "$KERNEL_TOP/common-kernel/kernel-dtb-$device"
+                obj="$OUT_OBJ_DEST/kernel-dtb-$device"
+                [ "$build_directory" ] && obj="$OUT_OBJ_DEST/kernel"
+                cp "$KERNEL_TMP/arch/arm64/boot/Image.gz-dtb" "$obj"
```

Comment:
If $OUT_OBJ_DEST is defined, destination directory will become .../out/.../obj/kernel/kernel/ (and that is an already existing directory)
In short, please always copy kernel only to "$KERNEL_TOP/common-kernel/kernel-dtb-$device", as this is the main functionality of this script.
Nothing looks for something called `kernel` (or `kernel-dtb-$device`) under obj/.

Reply:
It is always defined but with the default path if just $OUT_OBJ_DEST is modified that is used but when build_directory was set the kernel is copied to out_obj_dest with no $device.
When invoking the script like normal then nothing changes.